tortoise/verifying_tortoise.go

```diff
@@ -824,13 +822,6 @@ func (t *turtle) verifyLayers(ctx context.Context) error {
 candidateLayerLoop:
 	for candidateLayerID := t.Verified.Add(1); candidateLayerID.Before(t.Last); candidateLayerID = candidateLayerID.Add(1) {
 		logger := logger.WithFields(log.FieldNamed("candidate_layer", candidateLayerID))
-
```

Comment:
why removed?

Reply:
i removed it because i cannot figure out how this could happen. self-healing is running in the same thread. no other thread should be able to change t.Verified. do you read differently?